cmd/alertmanager/main.go

```diff
@@ -180,10 +182,14 @@ func run() int {
 	}
 
 	var (
-		configFile      = kingpin.Flag("config.file", "Alertmanager configuration file name.").Default("alertmanager.yml").String()
-		dataDir         = kingpin.Flag("storage.path", "Base path for data storage.").Default("data/").String()
-		retention       = kingpin.Flag("data.retention", "How long to keep data for.").Default("120h").Duration()
-		alertGCInterval = kingpin.Flag("alerts.gc-interval", "Interval between alert GC.").Default("30m").Duration()
+		configFile = kingpin.Flag("config.file", "Alertmanager configuration file name.").Default("alertmanager.yml").String()
+		dataDir    = kingpin.Flag("storage.path", "Base path for data storage.").Default("data/").String()
+		retention  = kingpin.Flag("data.retention", "How long to keep data for.").Default("120h").Duration()
+
+		alertGCInterval      = kingpin.Flag("alerts.gc-interval", "Interval between alert GC.").Default("30m").Duration()
+		alertStorageProvider = kingpin.Flag("alerts.storage.provider", "Alert storage \"mem\" or \"etcd\".").Default("mem").String()
+		alertEtcdEndpoints   = kingpin.Flag("alerts.etcd.endpoint", "Etcd server used for alert storage(may be repeated for each in etcd cluster).").Default("localhost:2379").Strings()
+		alertEtcdPrefix      = kingpin.Flag("alerts.etcd.prefix", "String prefix for storing AM alerts in etc").Default("am/alerts-").String()
```

Comment:
Q: Does etcd prefix not start with a leading `/`? Just checking, since I'm not sure.

Reply:
Etcd key structures are flat.  Even the "/" separator doesn't imply foldering as of etcd v3.

Reply:
Ok thanks. Must have been a v2 vestige in my mind.